### PR TITLE
Update End User Resources 

### PR DESCRIPTION
**Pull request by reese-lee**
As part of our work in transitioning the OTel End User WG to the OTel End User SIG (see [REQUEST: New repository opentelemetry-end-user](https://github.com/open-telemetry/community/issues/2004)), this PR updates the end user resources pages:

* Updates Slack channel names
* Removes end user discussion group page 
* Updates with new resources 

TODOs:
[] Link to new SIG repo when created
[] Link to new SIG charter when created
[] Create page that lists upcoming panels and current surveys -- calendar? 
[] Link to the above page when available
[] Create blog post announcement about above changes
[] Create page or section about how end users can contribute, including by voting on issues

### CODE REVIEW

**Review by svrnm — APPROVED** [review]

**Comment by svrnm** [comment]
```
blog/2023/end-user-discussions-01/index.html
  target does not exist --- blog/2023/end-user-discussions-01/index.html --> /community/end-user/discussion-group/
community/end-user/index.html
  hitting --- community/end-user/index.html --> https://www.youtube.com/@otel-official
community/end-user/interviews-feedback/index.html
  hitting --- community/end-user/interviews-feedback/index.html --> https://www.youtube.com/playlist?list=PLVYDBkQ1TdywIl9xKEo5_u7zlwY38dW43
  Non-OK status: 404 --- community/end-user/interviews-feedback/index.html --> https://www.youtube.com/playlist?list=PLVYDBkQ1TdywIl9xKEo5_u7zlwY38dW43
blog/2023/end-user-discussions-02/index.html
  target does not exist --- blog/2023/end-user-discussions-02/index.html --> /community/end-user/discussion-group/
blog/2023/end-user-discussions-03/index.html
  target does not exist --- blog/2023/end-user-discussions-03/index.html --> /community/end-user/discussion-group/
blog/2023/euwg-future-of-observability-panel/index.html
  target does not exist --- blog/2023/euwg-future-of-observability-panel/index.html --> /community/end-user/discussion-group/
blog/2023/humans-of-otel/index.html
  target does not exist --- blog/2023/humans-of-otel/index.html --> /community/end-user/discussion-group/
community/end-user/otel-in-practice/index.html
  hitting --- community/end-user/otel-in-practice/index.html --> https://www.youtube.com/playlist?list=PLVYDBkQ1TdyxKgdGE4ThYLkNRCuLLYy9x
  Non-OK status: 404 --- community/end-user/otel-in-practice/index.html --> https://www.youtube.com/playlist?list=PLVYDBkQ1TdyxKgdGE4ThYLkNRCuLLYy9x
blog/2023/end-user-q-and-a-02/index.html
  target does not exist --- blog/2023/end-user-q-and-a-02/index.html --> /community/end-user/discussion-group/
blog/2022/end-user-discussion-group-apac/index.html
  target does not exist --- blog/2022/end-user-discussion-group-apac/index.html --> /community/end-user/discussion-group/
blog/2023/contributing-to-otel/index.html
  target does not exist --- blog/2023/contributing-to-otel/index.html --> /community/end-user/discussion-group/
blog/2023/end-user-discussions-04/index.html
  target does not exist --- blog/2023/end-user-discussions-04/index.html --> /community/end-user/discussion-group/
blog/2023/end-user-q-and-a-01/index.html
  target does not exist --- blog/2023/end-user-q-and-a-01/index.html --> /community/end-user/discussion-group/
blog/2023/end-user-q-and-a-04/index.html
  target does not exist --- blog/2023/end-user-q-and-a-04/index.html --> /community/end-user/discussion-group/
blog/2023/end-user-q-and-a-03/index.html
  target does not exist --- blog/2023/end-user-q-and-a-03/index.html --> /community/end-user/discussion-group/
========================================================================
✘✘✘ failed in 42.26636529s
14 errors in 600 documents
```

Removing that page breaks a few existing blog posts, we either need to redirect discussion-group somewhere or keep that page and say that there are no discussion groups anymore or something similar

File: content/en/community/end-user/_index.md [hunk 1]
```diff
@@ -1,20 +1,24 @@
 ---
 title: End-user Resources
```
Comment (theletterf) [inline content/en/community/end-user/_index.md:2]:
```suggestion
title: End-user resources
```

File: content/en/community/end-user/_index.md [hunk 2]
```diff
@@ -1,20 +1,24 @@
...
 linkTitle: End User
```
Comment (theletterf) [inline content/en/community/end-user/_index.md:3]:
```suggestion
linkTitle: End user
```

File: content/en/community/end-user/_index.md [hunk 3]
```diff
@@ -1,20 +1,24 @@
...
-The [OpenTelemetry End User Working Group][] has heard feedback from users who
-desire a vendor-agnostic space to discuss adopting OpenTelemetry, so we provide
-multiple ways for you to connect with other users of the project and share best
-practices:
+The
+[OpenTelemetry End User SIG](https://github.com/open-telemetry/community/tree/main/working-groups/end-user)
+has heard feedback from users who desire a vendor-agnostic space to discuss
+OpenTelemetry adoption and implementation, so we provide the following ways for
+you to connect with other users of the project, learn about OTel, and provide
+feedback about your OTel usage:
```
Comment (theletterf) [inline content/en/community/end-user/_index.md:12]: Sounds a bit like summoning a deity to me. :)

How about "If you're looking for a vendor-agnostic space to..."

I don't think users are concerned with how this came to happen, but at the end you could link the working group as the source of this initiative, or for learning more.

File: content/en/community/end-user/_index.md [hunk 4]
```diff
@@ -38,14 +42,11 @@ But here are some of the kinds of things we expect will be on the table:
 **Is this group only for OpenTelemetry end users?**
 
 No! Anyone is welcome to join and discuss their journey to observability. This
```
Comment (theletterf) [inline content/en/community/end-user/_index.md:44]:
```suggestion
Everybody is welcome to join and discuss their journey to observability. This
```

**Review by theletterf — APPROVED** [review]
Approved with some nits. Thanks!

**Review by danielgblanco — APPROVED** [review]
LGTM

File: content/en/community/end-user/discussion-group.md
Comment (danielgblanco) [inline content/en/community/end-user/discussion-group.md:1]: Have the APAC discussion group been notified about this? We have agreed to keep the APAC discussion group while stopping the EU and NA ones. Or do they also want to discontinue it?

Comment (reese-lee) [reply content/en/community/end-user/discussion-group.md:1]: Good point, and given the issues that arise with the removal of the page, I'm going to add it back in with some mods. Awaiting word from the APAC team.

File: content/en/community/end-user/_index.md (hunk 3 above)
Comment (reese-lee) [reply content/en/community/end-user/_index.md]: You're right, I felt it was a bit weird too but was going with what was already written... I rewrote it! Hopefully it reads better now, lmk what you think!

**Comment by reese-lee** [comment]
@svrnm Got it, I added that page back in with some mods.

**Comment by svrnm** [comment]
@reese-lee I pushed a commit that fixes the link issues: the main problem was with the feedback URL, since the regex pattern in `.htmltest.yml` (`^/community/end-user/feedback-survey/$`) did not match the relative URL, so I set it to absolute.

**Comment by svrnm** [comment]
```
Run if grep -B 1 -e '"StatusCode": 4' static/refcache.json; then
  "https://www.youtube.com/playlist": {
    "StatusCode": 404,
Run 'npx gulp prune' to remove 4xx entries from the refcache
Error: Process completed with exit code 1.
```

Ok, that's confusing, not sure where this comes from, something goes really wrong. @reese-lee would you mind rebasing this PR, maybe that makes this go away.

**Comment by reese-lee** [comment]
@svrnm I did the update with the rebase

File: content/en/community/end-user/slack-channel.md [hunk 5]
```diff
@@ -2,19 +2,28 @@
...
+  Join a Slack channel to discuss OpenTelemetry with other end users.
 weight: 40
 ---
 
-- Confirm your agreement with channel [Code of Conduct][] and reach out to Reese
-  Lee, Rynn Mancuso, or Adriana Villela on CNCF slack for an invite to
-  `#otel-endusers`.
+Previously, we had a private channel to provide a space for end users to discuss
+adoption and implementation issues while allowing them to maintain a layer of
+privacy and to meet compliance concerns. Now that the project and community have
```
Comment (ricardoamaro) [inline content/en/community/end-user/slack-channel.md:11]:
```suggestion
privacy and meet compliance concerns. Now that the project and community have
```

**Review by mhausenblas — APPROVED** [review]
LGTM

File: .htmltest.yml [hunk 6]
```diff
@@ -35,6 +35,8 @@ IgnoreURLs: # list of regexs of paths or URLs to be ignored
   - ^https://x.com
   # OTel Google calendar - curl returns 200, but the link checker gets a 401:
   - ^https://calendar.google.com/calendar/embed\?src=google.com_b79e3e90j7bbsa2n2p5an5lf60%40group.calendar.google.com
+  # YouTube playlists sometimes give a 404, although they give a 200 when accessed via browser:
+  - ^https://www.youtube.com/playlist\?list=
```
Comment (svrnm) [inline .htmltest.yml:39]: @chalin FYI

**Pull request merged**